crates/ibc-parlia-client-cw/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "ibc-parlia-client-cw"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = [ "cdylib", "rlib" ]
+
+[dependencies]
+cosmwasm-std	= { workspace = true }
+
+light-client	= { workspace = true }
+
+ibc-elc-client-cw = { path = "../ibc-elc-client-cw" }
+
+parlia-elc = { git = "https://github.com/datachainlab/parlia-elc", rev = "v0.3.8", version = "0.2.9", default-features = false, features = ["dev"] }
```

Comment:
Do we need `version` field for `parlia-elc`?

Reply:
fixed: https://github.com/datachainlab/ibc-elc-client-cw/pull/1/commits/edf55f95b7ea3f2d8d8c5dd2c7f00ca7b564d488